data/beaconrestapi/src/integration-test/resources/tech/pegasys/teku/beaconrestapi/beacon/migrated/schema/ProduceBlockV2Response.json

```diff
@@ -16,7 +16,7 @@
     },
     "version" : {
       "type" : "string",
-      "enum" : [ "phase0", "altair", "bellatrix" ]
+      "enum" : [ "phase0", "altair", "bellatrix", "capella" ]
```

Comment:
We probably don't want capella appearing here yet?

Reply:
I wasn't too worried about this, but I can take a stab at excluding it.. it's just more complicated with the way we do enumerations